tests/sentry/api/test_event_search.py

```diff
@@ -1199,21 +1176,21 @@ def test_field_alias_duration_expansion_with_brackets(self):
             ["avg", "transaction.duration", "avg_transaction_duration"],
             ["argMax", ["id", "timestamp"], "latest_event"],
             ["max", "timestamp", "last_seen"],
-            ["apdex(duration, 300)", None, "apdex"],
+            ["apdex(duration, 300)", None, "apdex_transaction_duration_300"],
```

Comment:
Do we need to update the front-end alias generation to reflect this change?

Reply:
Yes, these two alias functions will need to be kept in sync. Right now if I map old functions to new functions, I also map the results back to the original alias passed in by the frontend.